app/routes/console.php

```diff
@@ -119,29 +97,33 @@
 
     $this->info("Scraping vote collections for {$date}");
     $action->execute($term, $date);
-
-    $this->output->writeln('');
 })->describe('Scrape and save all vote collections for the given date and term.');
 
-Artisan::command('scrape:all {--term=} {--from=} {--to=}', function (
-    int $term,
-    string $from,
-    string $to
-) {
-    $period = Carbon::parse($from)->toPeriod($to);
-
-    Artisan::call('scrape:members', ['--term' => $term]);
-    Artisan::call('scrape:members-groups', ['--term' => $term]);
-    Artisan::call('scrape:members-info', ['--term' => $term]);
+Artisan::command('scrape:all {--term=}', function (int $term) {
+    // Artisan::call('scrape:members', ['--term' => $term]);
+    // Artisan::call('scrape:members-groups', ['--term' => $term]);
+    // Artisan::call('scrape:members-info', ['--term' => $term]);
 
-    $from = Carbon::parse($from);
-    $month = $from->month;
-    $year = $from->year;
     Artisan::call('scrape:sessions', [
-        '--year' => $year,
-        '--month' => $month,
+        '--month' => Carbon::now()->month,
+        '--year' => Carbon::now()->year,
     ]);
 
+    // Get the oldest session without associated votes and try to
+    // scrape voting lists and votes for the days of the session.
+    $session = Session::query()
+        ->whereDoesntHave('votes')
+        ->orderBy('start_date', 'asc')
+        ->first();
+
+    if (! $session) {
+        $this->info('Exiting as all sessions already have associated votes.');
+
```

Comment:
Können wir die rausnehmen?

Reply:
Hmm, warum? Ich fand das ganz hilfreich, in den Logs nachvollziehen zu können, dass der Command erfolgreich durchgelaufen ist, es aber einfach nichts zu tun gab.

Reply:
> Hmm, warum? Ich fand das ganz hilfreich, in den Logs nachvollziehen zu können, dass der Command erfolgreich durchgelaufen ist, es aber einfach nichts zu tun gab.

Voll! Ich meinte nur die leere Zeile 🙈🙃

Reply:
Ne, die Leerzeile macht der Formatter

Reply:
Aber wir können das machen, denke ich:

```php
return $this->info('...'); 
```